src/components/forms/forms.js

```diff
@@ -265,11 +285,11 @@ export const FormField = (config) => {
     h(FormFieldTitle, {
       title, hideTitle, description, helpText,
       required, formId: id, ariaLevel,
-      error
+      validation: getValidation()
     }),
     h(type.component, {
       ...config,
-      error, setError,
+      validation: getValidation(), setValidation: updateValidation,
```

Comment:
I am not sure exactly needs to change, but I do agree it does look a bit strange. I have to do this because there are two different possible sources of truth for what validation should be. As such, I have to wrap update and get around their own callbacks. That being said, I wrapped set validation in another lambda, which I think looks cleaner.

```js
    h(type.component, {
      ...config,
      validation: getValidation(),
      setValidation: (newValidation) => updateValidation(newValidation),
      formValue, setFormValue,
      required
    })
```

Let me know what you think.